src/python/PyWeek1/max_and_min_numbers.py

```diff
@@ -0,0 +1,19 @@
+def high_and_low(numbers):
+    numbers1 = [int(n) for n in numbers.split()]
```

Comment:
Naming (Numbers1,Numbers) not the best way to go with. I would use "splitNumbers" or "splitted".

Reply:
Also, "list1" is not needed and wrong naming for a string. You can return with the computed string.